### ISSUE TITLE
Documentation Request

### ISSUE THREAD

**Issue by TheOddPanda** [enocean-js/enocean-js#223]
Im currently trying to switch from the deprecated library of node-red-enocean to the new enocean.js library and I constantly run into functions that dont exist anymore. I cannot find any documentation on how it changed or anything alike. There are also no examples in your repository which makes it really hard to use the library. 
F.E. `node.debug(`Listener count (after remove): ${node.enocean.listenerCount("data")}`);` used to work in the old version, doesnt anymore in the new. 

` enocean.listen(this.serialPort);` used to work in the old version and doesnt work anymore.

**Comment by nabbl** [comment]
Yes this is really bad at the moment. The old library seemed to be much better and better documented. I had to switch because of nodejs and node-red upgrade and now I am stuck implementing certain features because there is nothing documented here.

**Comment by Holger-Will** [comment]
you can find some example flows here: https://flows.nodered.org/search?term=enocean

and sorry for that lack of documentation etc. 
it's some tough times right now and I had to take care of my business... I expect to find some more time for this project soon :-D

I'm willing to answer any question and help with the migration... taking PR for the documentation and examples ;-)
